ilastik/applets/multicut/multicutGui.py

```diff
@@ -305,13 +305,14 @@ def configure_operator_from_gui(self):
         # (e.g. the FreezeCache setting).
         self.configure_gui_from_operator()
 
-    def _handle_mulicut_update_clicked(self):
+    def _handle_multicut_update_clicked(self):
         def updateThread():
             """
             Temporarily unfreeze the cache and freeze it again after the views are finished rendering.
             """
             with self.set_updating():
                 self.topLevelOperatorView.FreezeCache.setValue(False)
+                self.topLevelOperatorView.FreezeClassifier.setValue(False)
```

Comment:
`FreezeClassifier` is a member of `OpEdgeTrainingWithMulticut`, but not `OpMulticut`.  This gui "mixin" class is designed to work with operators that expose the same slots as `OpMulticut` (at least), but shouldn't require all of the slots in `OpEdgeTrainingWithMulticut`.

So, ideally, this `FreezeClassifier` change should happen in `EdgeTrainingWithMulticutGui` or its base class, `EdgeTrainingGui`.  But the complicated updating logic here makes that slightly inconvenient.  I suppose you could move the updating logic into a member function, and then allow `EdgeTrainingWithMulticutGui` to override it:

```python
    def _handle_mulicut_update_clicked(self):
        def updateThread():
            """
            Temporarily unfreeze the cache and freeze it again after the views are finished rendering.
            """
            with self.set_updating():
                self._update_multicut_views()

        self.getLayerByName("Multicut Edges").visible = True
        # self.getLayerByName("Multicut Segmentation").visible = True
        th = threading.Thread(target=updateThread)
        th.start()

    def _update_multicut_views(self):
        self.topLevelOperatorView.FreezeCache.setValue(False)
        self.update_button.setEnabled(False)
        self.live_multicut_button.setEnabled(False)

        # This is hacky, but for now it's the only way to do it.
        # We need to make sure the rendering thread has actually seen that the cache
        # has been updated before we ask it to wait for all views to be 100% rendered.
        # If we don't wait, it might complete too soon (with the old data).
        ndim = len(self.topLevelOperatorView.Output.meta.shape)
        self.topLevelOperatorView.Output((0,) * ndim, (1,) * ndim).wait()

        # Wait for the image to be rendered into all three image views
        for imgView in self.editor.imageViews:
            if imgView.isVisible():
                imgView.scene().joinRenderingAllTiles()
        self.topLevelOperatorView.FreezeCache.setValue(True)
        self.update_button.setEnabled(True)
        self.live_multicut_button.setEnabled(True)
```

Then, in `EdgeTrainingWithMulticutGui`:

```python
    def _update_multicut_views(self):
        self.topLevelOperatorView.FreezeClassifier.setValue(False)
        super()._update_multicut_views()
        self.topLevelOperatorView.FreezeClassifier.setValue(True)
```

I wouldn't call that elegant, but I think it will work.